.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,17 +1,25 @@
+<!-- 
 Thanks for your contribution!
 
-Please fill out the following template with details about your pull request:
+Please fill out the following template with details about your pull request: 
+-->
 
-**Related Issue**
+
+**Related Issue:**
+<!-- 
 Please provide the related issue #. Note that in most cases, you should only be opening a pull request that implements a solution discussed in an issue. See our contribution guidelines for more info
 
```

Comment:
Looks like there should be a --> here :)

Reply:
Good catch!